modules/apps/asset/asset-categories-admin-web/src/main/java/com/liferay/asset/categories/admin/web/internal/exportimport/data/handler/AssetCategoryStagedModelDataHandler.java

```diff
@@ -281,8 +281,8 @@ protected void doImportStagedModel(
 		categoryUuids.put(category.getUuid(), importedCategory.getUuid());
 
 		portletDataContext.importPermissions(
-			AssetCategory.class, category.getCategoryId(),
-			importedCategory.getCategoryId());
+			AssetCategory.class, Long.valueOf(category.getCategoryId()),
```

Comment:
Hi @dantewang @yuhai ,
You actually do not need to do this convert, it should autobox. As long as you removed the deprecate method, this is going to link to the non-deprecate one.
Please fix all the changes like this and resend.

Thanks.
Tina.